package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-redux-epic",
-  "version": "0.3.1",
+  "version": "1.0.0",
```

Comment:
You shouldn't update the package version. The package maintainer is always responsible for this and it's unrelated to this PR.

Reply:
Oh my bad, wasn't aware of that will keep that in mind in future.